core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -247,6 +253,10 @@
 
     private static final boolean DEFAULT_USE_HTTP2_PREFACE = getBoolean("defaultUseHttp2Preface", true);
     private static final boolean DEFAULT_USE_HTTP1_PIPELINING = getBoolean("defaultUseHttp1Pipelining", false);
+    private static final boolean DEFAULT_USE_HTTP2_PING_ON_IDLE_CONNECTION =
+            getBoolean("defaultUseHttp2PingOnIdleConnection", false);
```

Comment:
I don't think we need `default` for any of the new flags - our request / response timeout we do since a user can control them at runtime too but these I think are only controllable by flags.